judo-ui-react/src/main/resources/actor/src/utilities/table.ts.hbs

```diff
@@ -33,6 +33,18 @@ export interface SidekickComponentProps<T> {
   data: T[];
 }
 
+export interface ToolBarActionProps {
+  name: string;
+  id: string;
+  startIcon?: string | undefined;
+  variant: any;
+  onClick?: any;
+  hiddenBy: boolean,
```

Comment:
It's a good practice to always define bool types as optional, because internally checking for `=== true` or `=== false` is easy, and in certain cases reduces the number of parameters on the calling side we are using "defaults".

Reply:
[f51c46b](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/365/commits/f51c46bc6ce006871f020c8d52167c195794f251)